### DISCUSSION TITLE
Additional Data Point

### DISCUSSION THREAD

**Discussion by aabidDevit** [Abhinandan-Kushwaha/react-native-gifted-charts#948] [Q&A]
I am getting addtional data point below graph. How to avoid this?
![Screenshot 2024-12-30 at 3 07 30 PM](https://github.com/user-attachments/assets/10a0f7d2-64f0-4253-a4da-fe3a74d7f512)

**Comment by Abhinandan-Kushwaha** [comment]
were you able to get a solution @aabidDevit

**Reply by aabidDevit** [reply]
Yes @Abhinandan-Kushwaha ,  pointerConfig={{showPointerStrip: false}} worked for me. Thanks!
